### PR TITLE
[RW-7278][risk-low] Remove billing buffer, drop db table, delete related cron jobs

### PR DESCRIPTION
**Pull request by yonghaoy**
Description:

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: api/db/changelog/db.changelog-179-drop-billing-project-buffer-entry-table.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+<changeSet author="calbach" id="changelog-179-drop-billing-project-buffer-entry-table">
```
Comment (calbach) [inline api/db/changelog/db.changelog-179-drop-billing-project-buffer-entry-table.xml:7]: nit: author

File: api/db/changelog/db.changelog-179-drop-billing-project-buffer-entry-table.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    <dropTable tableName="billing_project_buffer_entry"/>
```
Comment (calbach) [inline api/db/changelog/db.changelog-179-drop-billing-project-buffer-entry-table.xml:8]: I have had to do a couple of bulk cleanups in lower environments where this table has been very useful - e.g. when we accidentally created a bunch of spurious projects that we thought were failures. I'm slightly worried that we may have a need for it later, if we discover some kind of regression / bug. Thoughts on leaving this around for a bit?

**Review by calbach — APPROVED** [review]
Massive code deletion - thank you! Just one comment about possibly keeping the table.

File: api/db/changelog/db.changelog-179-drop-billing-project-buffer-entry-table.xml (hunk 2 above)
Comment (yonghaoy) [reply api/db/changelog/db.changelog-179-drop-billing-project-buffer-entry-table.xml:8]: that's good point. I also realized we may want to delete those available projects, that also needs this table

**Pull request merged**